### ISSUE TITLE
SVG Rendring Error. With mathjax (tex2svg) generated SVGs are not rendered.

### ISSUE THREAD

**Issue by mogoh** [vivliostyle/vivliostyle.js#1135]
A SVG generated by tex2svg is not rendered by vivliostyle.
Here is an example:

```html
<!DOCTYPE html>
<html lang="en-Us">
<head>
<meta charset="utf-8">
<title>Examples</title>
</head>

<body>
Before SVG
<svg
xmlns:xlink="http://www.w3.org/1999/xlink"
width="9.173ex"
height="2.343ex"
style="vertical-align: -0.505ex;"
viewBox="0 -791.3 3949.5 1008.6"
role="img"
focusable="false"
xmlns="http://www.w3.org/2000/svg"
>
<defs>
<path stroke-width="1" id="E1-LATINMODERNNORMAL-1D44E" d="M498 143c0 0 -13 -63 -30 -99c-16 -32 -39 -55 -74 -55c-48 0 -83 33 -91 75c-60 -71 -110 -75 -130 -75c-78 0 -133 66 -133 160c0 146 124 293 241 293c45 0 74 -27 92 -64c3 22 18 44 42 44c17 0 29 -10 29 -27c0 -4 0 -6 -7 -34l-36 -140l-22 -90 c-11 -44 -13 -52 -13 -74c0 -20 3 -46 30 -46c41 0 59 59 76 124c3 14 4 18 14 18c3 0 12 0 12 -10zM361 332c0 6 -14 88 -79 88c-40 0 -85 -37 -116 -96c-23 -46 -55 -169 -55 -219c0 -39 14 -94 64 -94c28 0 69 16 113 71c15 17 15 19 20 37l50 196c1 5 3 11 3 17Z"></path>
<path stroke-width="1" id="E1-LATINMODERNMAIN-2B" d="M722 250c0 -11 -9 -20 -20 -20h-293v-293c0 -11 -9 -20 -20 -20s-20 9 -20 20v293h-293c-11 0 -20 9 -20 20s9 20 20 20h293v293c0 11 9 20 20 20s20 -9 20 -20v-293h293c11 0 20 -9 20 -20Z"></path>
<path stroke-width="1" id="E1-LATINMODERNNORMAL-1D44F" d="M415 282c0 -144 -123 -293 -241 -293c-74 0 -127 62 -127 157c0 35 4 51 16 101l82 326c5 21 14 55 14 62c0 10 -2 17 -50 17c-15 0 -24 0 -24 12c0 18 11 19 19 20c26 2 99 10 122 10c13 0 13 -11 13 -11l-74 -301c30 31 71 60 117 60c80 0 133 -69 133 -160zM343 326 c0 64 -27 94 -63 94c-26 0 -71 -15 -120 -80c-9 -11 -9 -13 -15 -35l-22 -92c-16 -63 -16 -82 -16 -101c0 -74 33 -101 67 -101c39 0 85 36 118 103c18 38 51 153 51 212Z"></path>
<path stroke-width="1" id="E1-LATINMODERNMAIN-3D" d="M722 347c0 -11 -9 -20 -20 -20h-626c-11 0 -20 9 -20 20s9 20 20 20h626c11 0 20 -9 20 -20zM722 153c0 -11 -9 -20 -20 -20h-626c-11 0 -20 9 -20 20s9 20 20 20h626c11 0 20 -9 20 -20Z"></path>
<path stroke-width="1" id="E1-LATINMODERNNORMAL-1D450" d="M430 107c0 -12 -84 -118 -227 -118c-104 0 -162 79 -162 169c0 141 133 284 268 284c71 0 118 -37 118 -86c0 -40 -27 -64 -56 -64c-19 0 -37 11 -37 35c0 7 2 24 18 39c14 14 28 14 44 14c-14 27 -52 40 -86 40c-55 0 -110 -43 -141 -100c-34 -62 -54 -159 -54 -200 c0 -60 27 -109 90 -109c12 0 121 0 200 99c6 8 8 10 13 10c6 0 12 -7 12 -13Z"></path>
</defs>
<g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)">
<use xlink:href="#E1-LATINMODERNNORMAL-1D44E" x="0" y="0"></use>
<use xlink:href="#E1-LATINMODERNMAIN-2B" x="751" y="0"></use>
<use xlink:href="#E1-LATINMODERNNORMAL-1D44F" x="1752" y="0"></use>
<use xlink:href="#E1-LATINMODERNMAIN-3D" x="2459" y="0"></use>
<use xlink:href="#E1-LATINMODERNNORMAL-1D450" x="3516" y="0"></use>
</g>
</svg>
After SVG
</body>
```

The HTML is validated by: https://validator.w3.org/nu/#textarea

This is what gets rendered:

![Screenshot_20230301_135436](https://user-images.githubusercontent.com/580975/222145225-eb75f7a6-daa7-496b-9c2e-913294e0e816.png)

**Comment by MurakamiShinyu** [comment]
@mogoh Thanks for reporting the bug. I confirmed the problem, that vivliostye.js modifies id values in SVG but does not modify corresponding href values in the same way.

**Comment by MurakamiShinyu** [comment]
SVG `<use>` element with `xlink:href` attribute caused this problem (no problem with `href`; Note: `xlink:href` is deprecated in SVG 2, see: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href).

---

Fixed in #1139.

Test: https://vivliostyle-git-fix-issue1135-vivliostyle.vercel.app/#src=https://gist.githack.com/MurakamiShinyu/0f6e6394f853d6d5cf0bafb972935463/raw/issue1135test.html

Screenshot:
![Screenshot 2023-03-10 at 11 59 29](https://user-images.githubusercontent.com/3324737/224212277-1956a27e-570c-431e-a7cc-1669deb1c544.png)

**Comment by mogoh** [comment]
Tank you for your great work!
